src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -262,85 +157,6 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
       .expectStatus().isOk
       .expectBody()
       .jsonPath("$.draftAdjudication.id").isNumber
-      .jsonPath("$.draftAdjudication.prisonerNumber").isEqualTo(testAdjudication.prisonerNumber)
-      .jsonPath("$.draftAdjudication.incidentDetails.dateTimeOfIncident").isEqualTo("2010-11-12T10:00:00")
-      .jsonPath("$.draftAdjudication.incidentDetails.handoverDeadline").isEqualTo("2010-11-14T10:00:00")
-      .jsonPath("$.draftAdjudication.incidentDetails.locationId").isEqualTo(3)
-  }
-
-  @Test
-  fun `edit offence details for the draft adjudication`() {
```

Comment:
A minor thing, but this one actually had a better name and actually edited the statement..

The one we have left tests something specific and isn't in keeping with the rest of the int tests. You could change the name of the other, but it really ought to test the statement been changed because that's the important bit.